process/block/preprocess/transactions.go

```diff
@@ -954,6 +968,16 @@ func (txs *transactions) createAndProcessMiniBlocksFromMe(
 			continue
 		}
 
+		if txs.balanceComputation.HasAddressBalanceSet(tx.GetSndAddr()) {
+			ok = txs.balanceComputation.SubBalanceFromAddress(tx.GetSndAddr(), txTotalCost)
+			if !ok {
+				log.Error("createAndProcessMiniBlocksFromMe.SubBalanceFromAddress",
```

Comment:
This should really not happen :)

Reply:
Yep, this is why I put this message with log level error